ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -35,6 +46,88 @@ def _set_base_span_tags(
             else:
                 span.set_tag_str(API_KEY, api_key)
 
+    def llmobs_set_tags(
+        self,
+        resp: Any,
+        span: Span,
+        args: List[Any],
+        kwargs: Dict[str, Any],
+        err: Optional[Any] = None,
+    ) -> None:
+        """Extract prompt/response tags from a completion and set them as temporary "_ml_obs.*" tags."""
+        if not self.llmobs_enabled:
+            return
+
+        parameters = {
+            "temperature": float(kwargs.get("temperature", 1.0)),
+            "max_tokens": float(kwargs.get("max_tokens", 0)),
+        }
+        messages = get_argument_value(args, kwargs, 0, "messages")
+        input_messages = self._extract_input_message(messages)
```

Comment:
We'll need to add system prompts as a message.

Reply:
done